.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,108 @@
+name: Bug Report [Recommended Template]
```

```suggestion
name: Bug report [Recommended Template]
```
very nit, but for consistency with other template